docs/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "MUI Team",
   "license": "MIT",
   "scripts": {
-    "build": "rimraf ./export && cross-env NODE_ENV=production next build --profile && pnpm build-sw",
+    "build": "rimraf ./export && node ../packages/x-charts-vendor/scripts/build.js && cross-env NODE_ENV=production next build --profile && pnpm build-sw",
```

Comment:
Build the vendor package otherwise netlify has no access to it

Reply:
This currently means all our users and devs need to build the lib before we can run `pnpm docs:dev`, else docs can't find the files. We probably need to run it on the `dev` command as well 🤔

Reply:
I assumed that once merged, everybody would run the build, and then it's done since all the generated files are in .gitignore

But effectively it's not friendly for external contributors. Maybe adding a parameter such that `build.js chached` check if some files already exist. if yes it just skip the build. Such that when running dev, we don't lose time generating this package

Reply:
Yeah, maybe check if the entrypoints are present and run the script otherwise?